sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala

```diff
@@ -199,4 +205,25 @@ object OrcUtils extends Logging {
       s"map<${orcTypeDescriptionString(m.keyType)},${orcTypeDescriptionString(m.valueType)}>"
     case _ => dt.catalogString
   }
+
+  /**
+   * @return Returns the result schema string based on the canPruneCols flag.
+   *         resultSchemaString will be created using resultsSchema in case of
+   *         canPruneCols is true and for canPruneCols as false value
+   *         resultSchemaString will be created using the actual dataSchema.
```

Comment:
This description is not clear enough. This utility function also changed the value of `conf`. We need to document it. 

@SaurabhChawla100 Could you submit a follow-up PR to improve the description?

Reply:
@gatorsmile - This is the new helper method that we have added as the part of this PR

sure I Will update the description of PR . Shall I raised the PR against the new Jira or with this same jira . Since this Jira is already resolved

Reply:
> Shall I raised the PR against the new Jira or with this same jira

Its okay to refer to this JIRA ticket. Then, please add `[FOLLOWUP]` in the PR title.